extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaLicense.scss

```diff
@@ -0,0 +1,11 @@
+@import "skins/shared/color";
+@import "color";
+
+.ve-ui-wikia-license {
+	background-color: rgba($color-black, .1);
+	border-top: 1px solid $color-page-border;
+	color: $color-ve-text-secondary;
+	font-size: 0.85em;
```

Comment:
https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#leadingzero …but, as it's a dirty hack, I can live with a leading zero. ;)